tests/test_parallelization.py

```diff
@@ -8,84 +8,50 @@
-            {"//": "Parallel Rz(1)"},
-            {
-                "qop": "RZ",
-                "angles": [[1.0], "pi"],
-                "args": [["q", 0], ["q", 1], ["q", 2], ["q", 3]],
-            },
-            {"//": "Parallel PhasedX(0.5, 0.5)"},
-            {
-                "qop": "R1XY",
-                "angles": [[0.5, 0.5], "pi"],
-                "args": [["q", 0], ["q", 1], ["q", 2], ["q", 3]],
-            },
-            {"//": "Parallel RZZ"},
-            {
-                "block": "qparallel",
-                "ops": [
-                    {
-                        "qop": "RZZ",
-                        "angles": [[0.125], "pi"],
-                        "args": [[["q", 0], ["q", 1]]],
-                    },
-                    {
-                        "qop": "RZZ",
-                        "angles": [[1.0], "pi"],
-                        "args": [[["q", 2], ["q", 3]]],
-                    },
-                ],
-            },
-            {"mop": "Transport", "duration": [0.0, "ms"]},
-            {
-                "qop": "Measure",
-                "args": [["q", 0], ["q", 1], ["q", 2], ["q", 3]],
-                "returns": [["c", 0], ["c", 1], ["c", 2], ["c", 3]],
-            },
-            {"mop": "Transport", "duration": [0.0, "ms"]},
-        ],
-    }
-
-    assert actual["ops"][7]["block"] == "qparallel"
-    for op in expected["ops"][7]["ops"]:
-        assert op in actual["ops"][7]["ops"]
-
-    act_meas_op = actual["ops"][9]
-    assert act_meas_op["qop"] == "Measure"
-    assert sorted(act_meas_op["args"]) == expected["ops"][9]["args"]
-    assert sorted(act_meas_op["returns"]) == expected["ops"][9]["returns"]
+    phir = get_phir_json(QasmFile.parallelization_test, rebase=True)
+
+    # Make sure The parallel RZ and R1XY gates have the correct arguments
+    parallel_rz1 = phir["ops"][3]
+    assert parallel_rz1["qop"] == "RZ"
+    qubits = [["q", 0], ["q", 1], ["q", 2], ["q", 3]]
+    for qubit in qubits:
+        assert qubit in parallel_rz1["args"]
+    parallel_phasedx = phir["ops"][5]
+    assert parallel_phasedx["qop"] == "R1XY"
+    for qubit in qubits:
+        assert qubit in parallel_phasedx["args"]
+
+    # Make sure the parallel block is properly formatted
+    block = phir["ops"][7]
+    assert block["block"] == "qparallel"
+    assert len(block["ops"]) == 2
+    qop0 = block["ops"][0]
+    qop1 = block["ops"][1]
+    assert qop0["qop"] == qop1["qop"] == "RZZ"
+
+    # Make sure the ops within the parallel block have the correct arguments
+    assert len(qop0["args"][0]) == len(qop1["args"][0]) == 2
+    q01_first = (["q", 0] in qop0["args"][0]) and (["q", 1] in qop0["args"][0])
+    q01_second = (["q", 0] in qop1["args"][0]) and (["q", 1] in qop1["args"][0])
+    q23_first = (["q", 2] in qop0["args"][0]) and (["q", 3] in qop0["args"][0])
+    q23_second = (["q", 2] in qop1["args"][0]) and (["q", 3] in qop1["args"][0])
```

Comment:
Because of their visual indentation, these are still hard to read, consider using `fst`, `snd` in the var names.

Reply:
@Asa-Kosto-QTM could you make these suggested changes in all places?